src/client/App.js

```diff
@@ -1,19 +1,71 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import TestComponent from "./components/TestComponent/TestComponent";
+import Home from "./components/Home/Home";
+import Meals from "./components/Meals/Meals";
+import AddReservation from "./components/AddReservation/AddReservation";
+import Createmeal from "./components/Meals/CreateMeal";
+import Mealreviews from "./components/Meals/MealReviews";
+import Navbar from "./components/Navbar/Navbar";
+import Footer from "./components/Footer/Footer";
+import About from "./components/About/About";
+
 
 function App() {
+
+  const [meals, setMeals] = React.useState([]);
+  const [reviews, setReviews] = React.useState([]);
+  const [stars, setStars] = React.useState([]);
+
+  React.useEffect(() => {
+    fetch("/api/meals")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        setMeals(data);
+      });
+    fetch("/api/reviews")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        setReviews(data);
+      });
+    fetch("/api/meals?stars=true")
+      .then((response) => response.json())
+      .then((data) => {
+        console.log(data);
+        setStars(data);
+      });
```

Comment:
This is also a bit backend related, but what I would do is that I would return the avg stars inside with the meals object as a property. That way you avoid having to always keep in sync the meals and stars, and try to find which meal from which start.meal_id that you're doing in your components.

Reply:
True .. its good idea.. Thanks for suggestion